packages/flutter_test/lib/src/controller.dart

```diff
@@ -143,7 +143,7 @@ class WidgetController {
   ///
   /// * Use [firstState] if you expect to match several states but only want the first.
   /// * Use [stateList] if you expect to match several states and want all of them.
-  T state<T extends State>(Finder finder) {
+  T state<T extends State>(Finder finder) { // ignore: strong_mode_not_instantiated_bound
```

Comment:
There is a breaking change in the latest dev release to make strong mode be more conservative about guessing default type arguments when code leaves off a type argument.  Here, `State` is defined as

```dart
State<T extends StatefulWidget>
```

Previously, strong mode would just guess that you meant `State<StatefulWidget>` when you wrote `State`.  Writing this explicitly here will fix the error.

This error is primarily aimed at eliminating cases where we simply can't guess a reasonable answer, e.g.

```
class Foo<T extends Foo<T>> {}

Foo x; // What does Foo mean here?
```

There is some ongoing discussion as to whether to allow cases like `State` here where there is an answer we can guess.

@Hixie, @sethladd any thoughts about this?